### DISCUSSION TITLE
Subir programa del curso porfa :)

### DISCUSSION THREAD

**Discussion by maromero-21** [IIC2133-PUC/2024-1#2] [Ayuda] [answered]
Hola! Quería pedirles si es que pueden subir el programa del curso al Syllabus por favor. Que tengan buena tarde!

**Comment by benjavicente** [comment, marked as answer]
[Subido, aquí está](https://github.com/IIC2133-PUC/2024-1/blob/main/Administrativo/programa.pdf).
